tox.ini

```diff
@@ -1,15 +1,15 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,pypy,pypy3,coverage
+    flake8,py27,py34,py35,py36,py37,pypy,pypy3,coverage
 
 [testenv]
+usedevelop = true
```

Comment:
(I still think this should be a user-level choice to pass `tox --develop` on the command line, instead of the default, but I've no energy to argue.)

Reply:
TIL. `tox --develop` was the part I was missing to omit the  global `usedevelop = true`.